src/lesson3AbstractionSubject/Cat.java

```diff
@@ -0,0 +1,29 @@
+package lesson3AbstractionSubject;
```

Comment:
please respect the syntax rules for packages. there should be not any upper case letter